docs/examples/Irrigation.ipynb

```diff
@@ -2,37 +2,47 @@
  "cells": [
```

Comment:
<p>Can we add https://doi.org/10.5281/zenodo.1045339 link  to <code>standard dataset</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yes, great suggestion!</p>